[stable/redis] Add support for custom labels on redis services

bderickson opened the pull request:
The various redis services already supported custom annotations, but didn't allow for custom labels.

bderickson committed:
Add support for custom labels on redis services

Signed-off-by: Brian Derickson <brian.derickson@capacity.com>

bderickson committed:
bump redis chart version to 9.2.3

Signed-off-by: Brian Derickson <brian.derickson@capacity.com>

miguelaeh commented on stable/redis/Chart.yaml:
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: redis
-version: 9.2.2
+version: 9.2.3
```
Please bump the minor version as you are adding a new feature.

bderickson replied:
Done! Had to force push because I forgot to sign a couple of my commits and had to alter the commit messages.

miguelaeh reviewed (changes requested):
Thank you very much for the PR!!!
Please, add the new parameter to the `README.md`.

bderickson committed:
update redis to version 9.3.0

Adds value allowing for custom labels on services

Signed-off-by: Brian Derickson <brian.derickson@capacity.com>

bderickson committed:
add new labels values to redis readme

Signed-off-by: Brian Derickson <brian.derickson@capacity.com>

bderickson committed:
improve descriptions of service labels values for stable/redis

Specify that these labels are added to the default set (release, chart
name, etc), not a wholesale replacement of them.

Signed-off-by: Brian Derickson <brian.derickson@capacity.com>

miguelaeh commented on stable/redis/templates/redis-with-sentinel-svc.yaml:
```diff
@@ -8,6 +8,9 @@ metadata:
     chart: {{ template "redis.chart" . }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
+    {{- if .Values.master.service.labels }}
```
This should be the sentinel labels not the master.

bderickson replied:
Good catch! Fixed.

miguelaeh reviewed (changes requested):
Thank you very much for addressing the changes , please take a look this one too.

bderickson committed:
updated label value name used in redis sentinel service

Signed-off-by: Brian Derickson <brian.derickson@capacity.com>

miguelaeh reviewed (approved):
Thanks!!!!
